Fabric.Terminology.API/Extensions.ValueSetService.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Fabric.Terminology.API
+{
+    using System.Linq;
+
+    using Fabric.Terminology.API.Models;
+    using Fabric.Terminology.Domain;
+    using Fabric.Terminology.Domain.Models;
+    using Fabric.Terminology.Domain.Services;
+
+    public static partial class Extensions
+    {
+        public static Attempt<IValueSet> Create(this IValueSetService service, ValueSetCreationApiModel model)
```

Comment:
The `ValueSetCreationApiModel` is defined in the Fabric.Terminology.API project.  `IValueSetService` is defined in Fabric.Terminology.Domain which is a dependency for the API project.  I wanted to keep the API pocos separate from the Domain models for flexibility.